docs/wallet/wallet-extension.md

```diff
@@ -3,83 +3,162 @@ id: wallet-extension
+ 
+  ![wallet_extension_step9](/wallet/wallet-extension/wallet_extension_step9.png)
 
-Click on "*Send*." Add the wallet address, amount, and data.
+**Step 2:** Enter the address of the wallet you intend to send to and the amount. 
 
 ![maiar-defi-example-send](/wallet/wallet-extension/wallet_extension_step10.png)
 
-### Deposit to wallet
+*(Optional)* **Step 3**: Enter the data. This is a description of the transaction or any information you wish to pass through. 
 
-To deposit into a wallet, click on "*deposit*" and share your QR code or wallet.
+**Step 4:** Click on the _“Continue”_ button to complete the transaction.
+
+### Lock/unlock
+
+After 60minutes of being inactive, the extension automatically locks itself. You can unlock it at any time using your password. In addition, you can lock the extension manually, by clicking the **“lock”** icon in the header. 
+
+
+### Deposit to a wallet
+
+A deposit can be made to your wallet using the wallet extension. This feature allows you to share your QR code or wallet address to receive a token deposit. To get started:
+
+- Open up your Maiar Wallet extension.
+
+- Next, click on the _"*deposit*"_ and share your QR code or wallet address.
+ 
+  ![maiar-defi-example-deposit](/wallet/wallet-extension/wallet_extension_step11.png) 
 
-![maiar-defi-example-deposit](/wallet/wallet-extension/wallet_extension_step11.png)
 
 ### Transactions history
 
-The Maiar DeFi wallet has a record of every transaction that comes in and out of your wallet.
+On the wallet extension dashboard, the wallet records all transactions sent and received in your wallet. If you are a new user, it says "*No transactions found for this wallet*" until you make your first transaction.
 
-![maiar-defi-example-transaction](/wallet/wallet-extension/wallet_extension_step12.png)
+   ![wallet_extension_step9](/wallet/wallet-extension/wallet_extension_step9.png)
 
-### Networks
 
-In the settings section, there are network options to choose what network you are on. These networks include:
+### Networks
 
-1. Elrond Testnet
-2. Elrond Devnet
-3. Elrond Mainnet
+In the settings section on your extension dashboard, you can connect to the different networks provided by Elrond, such as the mainnet, testnet, and devnet.
 
 Choose either of these networks.
 
-*More information about these networks is contained subsequently in this documentation.*
+
+## Connecting the Maiar DeFi Wallet to Maiar Exchange App 
+
+You can now connect Maiar Exchange to the Maiar DeFi wallet in real-time. With this connection, you will be able to log in to the Maiar exchange using the Maiar DeFi wallet extension in a few steps.
+Follow these steps to proceed:
+
+**Step 1:** To get started, go to the [Maiar Exchange](https://maiar.exchange/) page on the right section of the page, click on the  _“connect”_ button.
+
+![wallet_extension_step17](/wallet/wallet-extension/wallet_extension_step17.PNG)
```

Comment:
can you please rename the images you have uploaded, for consistency? `PNG` -> `png`